### ISSUE TITLE
Aircon mode id between Homebridge and Tuya

### ISSUE THREAD

**Issue by donkeystudio** [prasad-edlabadka/homebridge-tuya-ir#69]
Hi, for Heater Cooler between Tuya and Homebridge, mode id mapping is as follow:

| Mode | Tuya | Homebridge |
| :-- | :-- | :-- |
| Cool | 0 | 2 |
| Heat | 1 | 1 |
| Auto | 2 | 0 |

I have fixed it in my fork [here](https://github.com/donkeystudio/homebridge-tuya-ir/commit/20a72d15f962644a5b409f9b528fad5303db1e6f). 

Cheers..

**Comment by prasad-edlabadka** [comment]
It's the same mapping
```
let command = 2;
if (val == this.platform.Characteristic.TargetHeaterCoolerState.COOL) command = 0;
if (val == this.platform.Characteristic.TargetHeaterCoolerState.HEAT) command = 1;
```
How is it different?

**Comment by donkeystudio** [comment]
Yup, that part is ok. It's not about the Set, it's about the initial Get

![image](https://user-images.githubusercontent.com/89248996/181000763-476489e2-f56d-45f8-a023-6ac7255ee133.jpeg)

**Comment by prasad-edlabadka** [comment]
Got it. Will fix this. Thanks for spotting.

**Comment by prasad-edlabadka** [comment]
Fixed in v1.0.6. Please check and let me know.

**Comment by donkeystudio** [comment]
Yes, just checked, I think that should do it. ^^
